extensions/blocks/logo-gallery/index.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import edit from './edit';
+import save from './save';
+import transforms from './transforms';
+import icon from './icon';
+
+const blockAttributes = {
```

Comment:
Note that `ids` is missing although it's here: https://github.com/Automattic/jetpack/pull/11986/files#diff-a115179d5ecac47ccefb51f6a9728bcbR64

Reply:
Agreed! Just pushed a commit that adds it.